pkg/pillar/cmd/diag/diag.go

```diff
@@ -851,27 +851,20 @@ func handleGlobalConfigModify(ctxArg interface{}, key string,
 		log.Infof("handleGlobalConfigModify: ignoring %s\n", key)
 		return
 	}
-	log.Infof("handleGlobalConfigModify for %s\n", key)
-	var gcp *types.GlobalConfig
-	debug, gcp = agentlog.HandleGlobalConfig(ctx.subGlobalConfig, agentName,
-		debugOverride)
+	var gcp *types.ConfigItemValueMap
+	debug, gcp = agentlog.HandleGlobalConfig(ctx.subGlobalConfig, agentName, debugOverride)
 	if gcp != nil {
```

Comment:
Why is sthis check needed anymore? What is the default value of ctx.configItemValueMap?